CONTRIBUTING.md

```diff
@@ -32,6 +32,16 @@ The project's test suite can be run with the python test runner, `green`
 green -vvv  ./tests/unit
 ```
 
+In order to run the integration test, one will need to have a zowe profile. If zowe cli is installed, the test profile lives in `~/.zowe/profiles/zomsf/Zowe.yaml`
+
+```
+host: zzow03.zowe.marist.cloud
+port: 10443
```

Comment:
This host and port is specific to the LFX mentorship.

For the documentation we should use a more generic host and port, something like `example.com` and `443`.